### PR TITLE
(MAIN-8391) Properly escape wiki name

### PR DESCRIPTION
**Pull request by Grunny**
Properly escape wiki name on Special:Newwikis and the UserIdentityBox.

/cc @macbre @TK-999

### CODE REVIEW

File: extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php [hunk 1]
```diff
@@ -195,17 +195,17 @@ function reallyDoQuery( $offset, $limit, $descending ) {
 	function formatRow( $row ) {
 		global $wgLang;
```
Comment (WikiaTech) [inline extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php]: MAJOR Remove this unused "$wgLang" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 23 issues
* CRITICAL 1 critical
* MAJOR 10 major
* MINOR 12 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. CRITICAL [SpecialNewWikis_body.php#L273](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L273): Only the first line of this 3-line block will be executed in a loop. The rest will execute only once. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2681)
1. MAJOR [SpecialNewWikis_body.php#L53](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L53): Remove this unused "$wgExternalSharedDB" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialNewWikis_body.php#L56](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L56): Remove this unused "$filename" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialNewWikis_body.php#L78](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L78): Remove this unused private "quote" method. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1144)
1. MAJOR [SpecialNewWikis_body.php#L108](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L108): Add curly braces around the nested statement(s). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialNewWikis_body.php#L241](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L241): Remove this unused "$wgRequest" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialNewWikis_body.php#L246](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L246): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1134)
1. MAJOR [SpecialNewWikis_body.php#L270](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L270): Add curly braces around the nested statement(s). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialNewWikis_body.php#L275](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L275): Add curly braces around the nested statement(s). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialNewWikis_body.php#L293](https://github.com/Wikia/app/blob/898464abe795144b3e6b9fae5230ff176c356817/extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php#L293): Add curly braces around the nested statement(s). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)

**Review by mszabo-wikia — APPROVED** [review]
LGTM.

**Comment by macbre** [comment]
:+1:

**Pull request merged**